Pipfile

```diff
@@ -0,0 +1,15 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pytest = "*"
+
```

Comment:
i believe you need to add `docker = "*"` here for tests to run in shell / isolated env (required for `molecule destroy`

Reply:
@cmharlow  So, I just tried this locally, and `molecule test` fails at the destroy step when I have `docker = "*"` in `[dev-packages]` and not in `[`packages]`.

Reply:
these are dev env requirements, not prod requirements, so lets figure it out.

Reply:
putting a PR against this to raise the discussion on handling molecule / ansible dev requirements vs prod (runtime?) requirements.

Reply:
No, I was running without the `--dev` flag so that was my issue. Agree that there are no runtime requirements for docker.